rosdep/base.yaml

```
@@ -6066,6 +6066,12 @@ sdl2:
   openembedded: [libsdl2@openembedded-core]
   rhel: [SDL2-devel]
   ubuntu: [libsdl2-dev]
+sdl2-image:
+  debian: [libsdl2-image-dev]
+  fedora: [SDL2_Image-devel]
+  rhel:
+    '7': [SDL2_Image-devel]
```

Ah, my fault, the `I` isn't supposed to be capitalized.
```suggestion
  fedora: [SDL2_image-devel]
  rhel:
    '7': [SDL2_image-devel]
```

---

Do you want me to rebase it, or you guys are going to squash merge?

---

We can squash it. Thanks!